gobblin-data-management/src/main/java/gobblin/data/management/copy/publisher/CopyEventSubmitterHelper.java

```diff
@@ -41,10 +41,12 @@
   public static final String SIZE_IN_BYTES = "SizeInBytes";
 
   static void submitSuccessfulDatasetPublish(EventSubmitter eventSubmitter,
-      CopyEntity.DatasetAndPartition datasetAndPartition, String originTimestamp, String upstreamTimestamp) {
-    SlaEventSubmitter.builder().eventSubmitter(eventSubmitter).eventName(DATASET_PUBLISHED_EVENT_NAME)
+      CopyEntity.DatasetAndPartition datasetAndPartition, String originTimestamp, String upstreamTimestamp,
+      String source, String destination, String azkabanUrl) {
```

Comment:
Be a bit more specific with the names: `source` -> `sourceUri`, `destination` -> `destinationUri`. The `azkabanUrl` is only valid for flows running on Azkaban, which is a minority, so it shouldn't be an argument of this method. Instead, you could put a map of additional metadata.